internal/server/routes.go

```diff
@@ -184,7 +184,7 @@ func add[Req, Res any](a *API, r *gin.RouterGroup, route route[Req, Res]) {
 			a.t.RouteEvent(c, route.path, Properties{"method": strings.ToLower(route.method)})
 		}
 
-		c.JSON(defaultResponseCodeForMethod(route.method), resp)
+		c.PureJSON(defaultResponseCodeForMethod(route.method), resp)
```

Comment:
This prevents html special-character escaping.

Reply:
PureJSON actually has another odd side effect of adding a newline character to the created JSON

Reply:
really don't think we can do this